sdk/src/third-party/noconflict.js

```diff
@@ -1,5 +1,4 @@
 /*!
```

Comment:
It would be good to take a look at how often `_` is used and how it is used to see if many that is an extra dependency that could be eliminated.

Reply:
@pc2459 if we really needed `_` how much effort do you think it would be to have something like webpack only inject the necessary bits and in such a way that we wouldn't need `noconflict.js` ?

Reply:
IIRC, it depends on how the package is coded and consumed. If we're nagging the UMD version of `_` that injects itself onto the window global, then sadness. But I'm pretty sure there're builds of it that don't, and that `_` can therefore be localised. It's been a while, lemme go look this up...

Reply:
Yeah, looks like in lodash's case the AMD version (which is what is pulled in by default) does the window declaration, and in that case you have to use the `noConflict` script - https://github.com/webpack/webpack/issues/4465 

Much like old F2, these things weren't coded particularly well for commonjs/es usage that would let Webpack easily prevent global pollution. There are a couple of other ways; the infamous `lodash-es` is an ES version, so we could try that and double check to see if global pollution has been avoided. Or the good ole `import { blah} from 'lodash'` will probably also help.